python/example_code/dynamodb/README.md

```diff
@@ -74,12 +67,139 @@ predictable performance with seamless scalability.*
+[Amazon DynamoDB developer guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/samples/moviedata.zip). 
+
+To start the scenario, navigate to the `batching` folder and run the following at a 
+command prompt:
+
+```
+python dynamo_batching.py
+```  
+
+### Accelerate reads with DAX
+
+Shows you how to use the Amazon DAX Client for Python to read items from a DynamoDB 
+table. Retrieval, query, and scan speeds are measured for both Boto3 and DAX clients 
+to show some of the performance advantages of using DAX.
+
+DAX is a DynamoDB-compatible caching service that provides fast in-memory performance 
+and high availability for applications that demand microsecond latency. For more
+information, see [In-Memory Acceleration with DynamoDB Accelerator (DAX)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DAX.html). 
+
+#### Running on your local computer
+
+When run on your local computer, only the Boto3 client works.
+
+Each file can be run separately at a command prompt. For example, create the
+table by running the following from a command prompt window.
+
+```commandline
+python 01-create-table.py
+```  
+
+#### Running on a DAX cluster
+
+To run the scripts with the DAX client, you must run them on an Amazon EC2 instance 
+within your VPC, as described in the Python sample application tutorial in the  
+[Developing with the DAX Client](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DAX.client.html) 
+section of the *Amazon DynamoDB Developer Guide*.
+
+The tutorial shows you how to set up the following additional resources:
+
+- An Amazon Virtual Private Cloud (Amazon VPC)
+- A DAX cluster set up in your VPC 
+- An Amazon Elastic Compute Cloud (Amazon EC2) instance running in your VPC with the
+  following installed:
+    - Python 3.7 or later
+    - Boto3 1.11.10 or later
+    - Amazon DAX Client for Python 1.1.7 or later
+
+On an EC2 instance, run the get item, query, and scan test scripts with the DAX client
+by specifying a DAX cluster endpoint as the first positional argument. For example,
+to run the query test script with the DAX client, run the following from a command 
+prompt window.
+
+```commandline
+python 04-query-test.py YOUR-CLUSTER-NAME.111111.clustercfg.dax.usw2.cache.amazonaws.com:8111
+```
+
+## Running the tests
+
+All tests use pytest and can be found in the `test` folder of each example.
+
+### Unit tests
+
+The unit tests in this module use the botocore Stubber, which captures requests before 
+they are sent to AWS and returns a mocked response so no charges are incurred on your 
+account.
+
+Run unit tests at a command prompt from the example subfolder by including the 
+`"not integ"` marker.
+
+```
+python -m pytest -m "not integ"
+```
+
+### Integration tests
+
+The integration tests in this module do make requests to AWS, creating and destroying
+resources in your account. Integration tests must be approached with caution and
+may incur charges.
```

Comment:
**Note:** The integration tests in this module make actual requests to AWS, which means they can create and destroy
resources in your account. These tests might also incur charges. Proceed with caution.

Reply:
done